File: grader.py
import sys

class Grader:
    def __init__(self):
        self.name = input("What is your name? ")
        self.assignment = input("What is the assignment's name? ")
        self.gradeStr = input ("What grade did you get? ")
        self.grade = int(float(self.gradeStr))

    def getGrade(self):
        if self.grade >= 100:
            return self.name + ", you got a perfect A+ in " + self.assignment + "!"
        elif self.grade >= 90 and self.grade < 100:
            return self.name + ", you got an A in " + self.assignment +"! Great job!"
        elif self.grade >= 80 and self.grade < 90:
            return self.name + ", you got a B in " + self.assignment +". Keep up the good work!"
        elif self.grade >= 70 and self.grade < 80:
            return self.name + ", you got a C in " + self.assignment +". Make sure you're checking for errors"
        elif self.grade >= 60 and self.grade < 70:
            return self.name + ", you got a D in " + self.assignment +". Please review the material again"
        else:
            return self.name + ", you got an F in " + self.assignment +", see me after class."

myGrader = Grader()
print(myGrader.getGrade())

File: grader.js
class Grader{
    constructor(){
        const readline = require("readline");
        const rl = readline.createInterface({
            input : process.stdin,
            output : process.stdout
        })
    
        rl.question("What is your name? ",(name)=>{
            rl.question("What is the assignment called? ",(assignment)=>{
                rl.question("What grade did you get? ",(grade)=>{
                    if(Number(grade) >= 100){
                        console.log(name + ", you got a perfect A+ in " + assignment +"!")
                    }
                    else if(Number(grade) >= 90 && Number(grade) < 100){
                        console.log(name + ", you got an A in" + assignment + "! Great job!")
                    }
                    else if(Number(grade) >= 80 && Number(grade) < 90){
                        console.log(name + ", you got a B in" + assignment + ". Keep up the good work!")
                    }
                    else if(Number(grade) >= 70 && Number(grade) < 80){
                        console.log(name + ", you got a C in" + assignment + ". Make sure you're checking for errors.")
                    }
                    else if(Number(grade) >= 60 && Number(grade) < 70){
                        console.log(name + ", you got a D in" + assignment + ". Please review the material again")
                    }
                    else{
                        console.log(name + ", you got an F in" + assignment + ". See me after class.")
                    }
                    rl.close()
                })
                rl.close()
            })
            rl.close()
        })
    }
}

myGrader = new Grader()